src/cluster/generated/proto/placementpb/placement.proto

```diff
@@ -93,3 +93,12 @@ message PlacementSnapshots {
   CompressMode compress_mode = 2;
   bytes compressed_placement = 3;
 }
+
+message Options {
+  OptionalBool is_sharded = 1;
```

Comment:
I would use `google.protobuf.BoolValue` here since you've already imported it as well (the `wrappers.proto`):
https://github.com/protocolbuffers/protobuf/blob/38f6e15e77a4e33a669d5d37efd87edfa15c7c04/src/google/protobuf/wrappers.proto#L103-L107